.github/DISCUSSION_TEMPLATE/request-help.yml

```diff
@@ -1,4 +1,13 @@
 body:
+  - type: dropdown
+    id: question-type
+    attributes:
+      label: What would you like help with?
+      options:
+        - 'I would like help with my configuration'
+        - 'I think I found a bug'
```

Comment:
Both these are phrased non-aggressively

Reply:
I like the general wording, makes it sound welcoming 👍